product/import_products_big_query_table.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main() {
+  // [START retail_import_products_from_big_query]
+
+  // Imports the Google Cloud client library.
+  const { ProductServiceClient } = require('@google-cloud/retail').v2;
+
+  const projectId = process.env['PROJECT_NUMBER'];
+
+  const datasetId = 'products';
+  const tableId = 'import_tutorial';
+  const dataSchema = 'product';
+
+  // Placement
+  const parent = `projects/${projectId}/locations/global/catalogs/default_catalog/branches/default_branch`
+
+  // The desired input location of the data.
+  const inputConfig = {
+    bigQuerySource: {
+      projectId,
+      datasetId,
+      tableId,
+      dataSchema
+    }
+  }
+
+  // The mode of reconciliation between existing products and the products to be imported.
+  const reconciliationMode = 2
+
+  // Instantiates a client.
+  const retailClient = new ProductServiceClient();
+
+  const callImportProducts = async () => {
+    // Construct request
+    const request = {
+      parent,
+      inputConfig,
+      reconciliationMode
+    };
+
+    // Run request
+    const [operation] = await retailClient.importProducts(request);
+    const [response] = await operation.promise();
+    console.log(response);
```

Comment:
please print out the operation response fields like this: 
    print("---import products operation is done---")
    print("---number of successfully imported products---")
    print(big_query_operation.metadata.success_count)
    print("---number of failures during the importing---")
    print(big_query_operation.metadata.failure_count)
    print("---operation result:---")
    print(big_query_operation.result())